shell/platform/linux/fl_renderer.cc

```diff
@@ -100,6 +103,136 @@ static void fl_renderer_unblock_main_thread(FlRenderer* self) {
   }
 }
 
+static void setup_shader(FlRenderer* self) {
+  FlRendererPrivate* priv = reinterpret_cast<FlRendererPrivate*>(
+      fl_renderer_get_instance_private(self));
+
+  GLuint vertex_shader = glCreateShader(GL_VERTEX_SHADER);
+  glShaderSource(vertex_shader, 1, &vertex_shader_src, nullptr);
+  glCompileShader(vertex_shader);
+  int vertex_compile_status;
```

Comment:
nit: here and below: I assume that `GLint` is defined as `int` consistently in every implementation but I thought we used `GLint` against GL APIs elsewhere.

Reply:
Agree, we should use the GL types here. Note this is in the existing code, not the change added in this PR.